search-commons/src/test/java/no/unit/nva/search/models/IndexDocumentTest.java

```diff
@@ -59,9 +58,8 @@ void shouldThrowExceptionWhenEventConsumptionAttributesDoNotContainDocumentIdent
     }
 
     private ObjectNode randomJsonObject() throws JsonProcessingException {
-        Publication publication = PublicationGenerator.publicationWithIdentifier();
-        String jsonString = objectMapper.writeValueAsString(publication);
-        return (ObjectNode) objectMapper.readTree(jsonString);
+        String json = randomJson();
```

Comment:
replacing the publication with a random json object.No tests needed to be updated